systemctl: add shutdown --show option

lnussel opened the pull request:
Shows the scheduled shutdown action and time if there's one.

lnussel commented:
Not sure what's the most obvious choice for an option here. show (-s), query (-q) or list (-l)? -q is often used for quiet though.

bluca commented:
Wouldn't a long option (`--show`) be more readable?

medhefgo commented:
shutdown is a legacy binary we provide for compatibilty. It should not be getting any new features.

bluca commented:
oh right, I missed that this was for the shutdown command rather than systemctl itself - then yes @medhefgo is right. Can this be added to systemctl reboot/poweroff/etc instead?

medhefgo commented:
> oh right, I missed that this was for the shutdown command rather than systemctl itself - then yes @medhefgo is right. Can this be added to systemctl reboot/poweroff/etc instead?

It's adding it for both. So the shutdown part just needs to be removed. Also needs update to manpage.

lnussel commented:
The long option probably is nicer indeed though

keszybz commented:
I think it is OK to add this. `shutdown` is a compat command, but effectively we're the new owners of it on Linux, since most people are using our implementation. But please call the option `--show`. This reduces the risk of a conflict with another implementation.

(FWIW, I don't think `shutdown` is going away. Similarly to the `3` and `5` options on the kernel command line, things which are convenient will be always used.)

But please add the man page entry.

lnussel commented:
Looks like  I went into a rabbit hole looking at the shutdown handling :/
Anyway, one thing at a time. Renamed to --show and added mention in man page.

bluca commented:
Please update shell-completion/ too

lnussel commented:
Looks like there is no existing shell completion for the shutdown command? Only systemctl is covered but the change doesn't impact systemctl in a user visible way.

keszybz commented on man/shutdown.xml:
```diff
@@ -125,6 +125,13 @@
+        <term><option>--show</option></term>
+
+        <listitem><para>Show a pending shutdown action and time if
+        there is any</para></listitem>
```
Missing '.'.

lnussel replied:
eagle eyes :+1:
